tests/ssl-opt.sh

```diff
@@ -1585,6 +1585,19 @@ CLI_OUT="cli_out.$$"
 PXY_OUT="pxy_out.$$"
 SESSION="session.$$"
 
+# If MBEDTLS_ENTROPY_C && MBEDTLS_ENTROPY_NV_SEED && !MBEDTLS_NO_PLATFORM_ENTROPY
+# create seedfile to fix test fail.
+requires_config_enabled MBEDTLS_ENTROPY_C
+requires_config_enabled MBEDTLS_ENTROPY_NV_SEED
+requires_config_disabled MBEDTLS_NO_PLATFORM_ENTROPY
+create_seedfile()
+{
+    [ -f $1 ] || dd if=/dev/urandom of=$1 bs=64 count=1 2>/dev/null
```

Comment:
It would be better to create the seedfile unconditionally. Testing whether it exists missing the case of having a build tree with a 32-byte seedfile (build without SHA-512), running tests, then rebuilding with a 64-byte seedfile (build with SHA-512). Then, after the rebuild, the seedfile will be too small.

A larger seedfile is acceptable, and the largest size that can be required is 64 bytes. That's why the test scripts unconditionally create a 64-byte seedfile.